src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs

```diff
@@ -69,6 +74,11 @@ public async Task<RefundClaimStatus> TryClaimRefundAsync(DepositDetails deposit,
             now = (ulong) latestBlock.Timestamp;
             if (!deposit.CanClaimRefund(now))
             {
+                var timeLeftToClaimRefund = deposit.GetTimeLeftToClaimRefund(now);
+                if (timeLeftToClaimRefund > 0)
+                {
+                    if (_logger.IsError) _logger.Info($"Time left to claim a refund: {timeLeftToClaimRefund} seconds.");
+                }
```

Comment:
this code is already added - should be in private method to not introduce duplicated code

Reply:
also was it meant to be if(_logger ISERROR) then Info?